modules/core/src/test/java/org/apache/ignite/cdc/CdcSelfTest.java

```diff
@@ -98,15 +97,12 @@
         return params;
     }
 
-    /** Consistent id. */
-    private UUID consistentId = UUID.randomUUID();
-
     /** {@inheritDoc} */
     @Override protected IgniteConfiguration getConfiguration(String igniteInstanceName) throws Exception {
         IgniteConfiguration cfg = super.getConfiguration(igniteInstanceName);
 
         if (specificConsistentId)
-            cfg.setConsistentId(consistentId);
+            cfg.setConsistentId(igniteInstanceName);
```

Comment:
Why this change?

Reply:
This helps to avoid setting a node consistent ID to new IgniteConfiguration instances passed to the CDC every time. 

Please see below that we no longer reuse the same IgniteConfiguration instance to run both Ignite and CDC.

Reply:
> This helps to avoid setting a node consistent ID to new IgniteConfiguration instances passed to the CDC every time.

Why you want to avoid it?

Reply:
Can we revert these changes? And keep only chenage related to pageSize definition?

Reply:
> Why you want to avoid it

To avoid meaningless copy-paste.

>Can we revert these changes? 
No we can't. The existing tests did not catch the bug this PR is intended to solve.